qt/widgets/common/src/QtJobRunner.cpp

```diff
@@ -6,7 +6,10 @@
 // SPDX - License - Identifier: GPL - 3.0 +
 
 #include "MantidQtWidgets/Common/QtJobRunner.h"
+#include "MantidAPI/AlgorithmRuntimeProps.h"
+#include "MantidAPI/IAlgorithm.h"
 #include "MantidQtWidgets/Common/BatchAlgorithmRunner.h"
+#include "MantidQtWidgets/Common/ConfiguredAlgorithm.h"
```

Comment:
Are `AlgorithmRuntimeProps.h` and `IAlgorithm.h` needed as imports if only usage for `IConfiguredAlgorithm_sptr ` has been added?